### ISSUE TITLE
Discussion of handling async calls

### ISSUE THREAD

**Issue by machuga** [shakacode/reactrails-in-reagent#1]
Is asynchrony via core.async something you think may be of interest or at least worth researching, or are you hoping to aim for callbacks/promises?

**Comment by justin808** [comment]
@machuga I'm guessing this should be similar to how we handle async calls in Redux/React for client side rendering.

**Comment by machuga** [comment]
Right. Core.async is a Clojure and ClojureScript wrapping layer around
asynchronous calls that allow them to look sync in nature kind of like the
new async features in es7. They'd still function like the other types under
the hood.
